packages/frontend/src/routes/TransferWizardWrapper.js

```diff
@@ -0,0 +1,204 @@
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>Welldone Wallet</p>
+                    </td>
+                    <td>
+                        <p><a target="blank" href="https://welldonestudio.io/">https://welldonestudio.io/</a></p>
+                    </td>
+                    <td>
+                        <p>A multi-chain extension wallet that gives you control over all your assets from a single platform.</p>
+                    </td>
+                </tr>
+            </Table>
+            <br />
+            <h3>How does the Wizard Work?</h3>
+            <p>The Transfer Wizard is designed to improve the security posture of your connected accounts before migrating them to a new wallet by rotating your keys, cleaning up old keys, and securely transferring your accounts to a new wallet. You can access the Wallet Transfer Wizard from the banner at the top of wallet.near.org. </p>
+            <Warning><b>Note</b>: The wizard does not support unfunded accounts (implicit accounts) or accounts using Ledger. You can import your Ledger accounts manually to a supported wallet.</Warning>
+            <hr />
+            <Table>
+                <tr>
+                    <Td>
+                        <p><b>Step 1. Secure Your Accounts</b></p>
+                        <p>The first recommended security step is to rotate your keys, generating a new 12-word recovery phrase. This process is similar to updating your password.</p>
+                        <ol type="1">
+                            <li>Generate a new recovery phrase</li>
+                            <li>
+                                Securely write down or save your new recovery phrase
+                                <ol type="a">
+                                    <li>Confirm the account it recovers</li>
+                                    <li>Don't confuse it with your old recovery phrase</li>
+                                </ol>
+                            </li>
+                            <li>Verify your new recovery phrase</li>
+                            <li>Keep your new phrase in a safe place</li>
+                        </ol>
+                        <Important>Note: Your recovery phrase grants full access to your account. Keep it in a safe place and never share it with anyone. Pagoda or the Near Foundation cannot help recover lost accounts or assets.</Important>
+                    </Td>
+                    <ImageTd>
+                        <ScreenshotImg alt="Secure Accounts" src={secureAccountsImg} />
+                    </ImageTd>
+                </tr>
+                <tr>
+                    <Td>
+                        <p><b>Step2. Clean Up Your Keys</b></p>
+                        <p>Next, NEAR wants to reduce the amount of apps that you&rsquo;ve shared keys with, similar to revoking access for third-parties. This may cause you to be disconnected from some apps.</p>
```

Comment:
> &rsquo;

There are a few different escape codes in here: `rsquo`, `nbsp`, and `amp`. Can you please double check that these are all rendering correctly and consistently with the rest of the text? Or just replace with plain characters 👍